packages/webapp/src/containers/saga.js

```diff
@@ -631,6 +631,7 @@ export function* clearOldFarmStateSaga() {
       'CustomAnimalTypes',
       'DefaultAnimalTypes', // needs to be cleared for KPI count
       'FarmAddon',
+      'IrrigationPrescriptions',
```

Comment:
Did not know about this - TODO for me make Jira ticket to investigate if everything is up to date here or could import tags instead.

Reply:
https://lite-farm.atlassian.net/browse/LF-4820